src/vuestic-theme/vuestic-components/va-slider/VaSlider.vue

```diff
@@ -45,6 +45,7 @@
           class="va-slider__container__handler"
```

Comment:
Visible selection on sliding element shouldn't trigger on click. That's already done for checkbox.